designer/server/test/fixtures/auth.js

```diff
@@ -7,18 +10,59 @@ export const auth = {
   strategy: 'azure-oidc',
   credentials: {
```

Comment:
@alexluckett I'm seeing type errors on this line, can you check?

>Type '{ provider: "azure-oidc"; refreshToken: string; token: string; }' is missing the following properties from type 'AuthCredentials<UserCredentials, AppCredentials>': query, idToken, expiresIn

i.e. We've removed **query**, **idToken**, **expiresIn** and not put them back

Happy to do a quick run through and fix a few bits?

Love how you've added `token.generate()`

Reply:
Previously our auth fixture was "this person has already signed in" with everything in place

So it might take some wrangling with types